src/wincng.c

```diff
@@ -236,6 +243,91 @@
 #define PKCS_RSA_PRIVATE_KEY ((LPCSTR)(size_t)43)
 #endif
 
+static int
+_libssh2_wincng_bignum_resize(_libssh2_bn* bn, ULONG length);
+
+int
+_libssh2_wincng_ecdsa_curve_type_from_name(IN const char *name,
+                                           OUT libssh2_curve_type *out_curve);
+
+int
+_libssh2_wincng_parse_ecdsa_privatekey(OUT _libssh2_wincng_ecdsa_key **key,
```

Comment:
If these aren't used outside this file, can we make them `static`?

Reply:
Also move them inside `#if LIBSSH2_ECDSA`, due to being ECDSA-specific?

Reply:
good point, done